### ISSUE TITLE
publish dev branch to npm

### ISSUE THREAD

**Issue by timoxley** [ractivejs/ractive#2672]
This has been raised before in #2620 and #2326, though these are a bit old and it seems the conclusion was that it wasn't clear how best to do this with npm.

The answer is to use `npm dist-tag` or `npm publish --tag`.

e.g.

```
> npm run build
> cd build
> npm publish --tag=dev
```

See:

* https://docs.npmjs.com/getting-started/using-tags
* https://docs.npmjs.com/cli/dist-tag

This could be rolled into a `package.json` script like:

```json
{
  "scripts": {
    "publish-dev": "npm run build && (cd build && npm publish --tag=dev)"
  }
}
```

```
> npm run publish-dev
```

**Comment by evs-chris** [comment]
This is on my list of things to do after 0.8 makes it out, and that is just pending time to do the release. I'm thinking of having a monotonically incrementing build number based on the next numbers, so the first one out after 0.8 would be 0.9-pre-build1 and tagged at edge.

I still am not sure if it would be better to automatically publish _every_ successful build from travis. My current feeling is that a giant flood of ractive dev builds on npm wouldn't really be a good thing, so a manual publish after significant changes would probably be best.

**Comment by JonDum** [comment]
@evs-chris Is this something someone else can take off your plate? You're already the lifeblood of ractive.

**Comment by evs-chris** [comment]
0.8.0 has been released! and there are already bug fixes out for it! :laughing: 

I have published 0.8.1-build-1 tagged as edge to npm, which means it is now available with both of those (0.8.1-build-1 and edge) versions on npm and what will be our new cdn approach https://unpkg.com/ractive@edge

I'll probably cut 0.8.1 in a week or so if nothing new pops up between here and there.
